assets/js/theme/cart.js

```diff
@@ -158,9 +167,7 @@ export default class Cart extends PageManager {
 
         utils.api.productAttributes.configureInCart(itemId, options, (err, response) => {
             modal.updateContent(response.content);
-            const $productOptionsContainer = $('[data-product-attributes-wrapper]', this.$modal);
-            const modalBodyReservedHeight = $productOptionsContainer.outerHeight();
-            $productOptionsContainer.css('height', modalBodyReservedHeight);
+            this.$modal.triggerHandler(ModalEvents.opened);
```

Comment:
I believe we do not need to trigger opened event because it will be triggered by lib when popup opened